src/showcase/Showcase.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+import { StyledShowcase, StyledShowcaseItem } from './Showcase.style'
+
+export type ShowcaseProps = Readonly<{
+  children: Array<React.ReactNode>
+}>
+
+export const Showcase = (props: ShowcaseProps): JSX.Element => {
+  const { children } = props
+
+  return (
+    <StyledShowcase>
+      {children.map((child, index) => {
```

Comment:
nit: You might want to filter falsy elements, as we often have the following pattern:

```
{condition && ( <Element/> )}
```

Reply:
```{children.filter(Boolean).map((child, index) => {```